service/get_invoice_details.js

```diff
@@ -146,9 +146,9 @@ module.exports = ({cache, check, invoice, network}, cbk) => {
     {
       const net = parsedInvoice.network.toUpperCase();
 
-      return getRoutes({
+      return getRouteToDestination({
         lnd,
-        cltv_delta: parsedInvoice.cltv_delta,
+        cltv_delta: parsedInvoice.timeout,
```

Comment:
I don't think this is right

Reply:
Ah, you are right. It should be `cltv_delta`